src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -172,22 +182,14 @@ protected void ProcessAssemblyAction (AssemblyDefinition assembly)
 				break;
 
 			case AssemblyAction.CopyUsed:
-				Annotations.SetAction (assembly, AssemblyAction.Copy);
-				goto case AssemblyAction.Copy;
+				AssemblyAction assemblyAction = AssemblyAction.Copy;
```

Comment:
If we are updating scopes in a copyused assembly, we shouldn't change its action to copy.